src/components/Button.js

```diff
@@ -3,7 +3,7 @@ import { createUseStyles } from 'react-jss';
 import { Link } from 'components';
 import classnames from 'classnames';
 
```

Comment:
It doesn't really matter for this PR, but I'd change the name of this component to be something else since it's not a button anymore but could also be a link.